components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -264,6 +269,16 @@ public Response authorize(@Context HttpServletRequest request, @Context HttpServ
         }
     }
 
+    private void setCommonAuthIdToRequest(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
Let's create a product issue to support the flow when the status is successful internally without any redirection and mark it a high priority.

Reply:
Created the issue https://github.com/wso2/product-is/issues/11065